styles/globals.css

```diff
@@ -1,3 +1,17 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+    h1 {
+        @apply text-3xl font-bold lg:text-5xl;
+    }
+
+    h2 {
+        @apply text-xl font-bold lg:text-3xl;
+    }
+
+    button {
+        @apply px-3 py-1 !bg-red-200 rounded-md shadow-red-200 shadow-sm hover:shadow-red-200 hover:shadow-md transition-shadow;
```

Comment:
yayy tailwind has colored shadows now

Reply:
Yep! I knew I had to get my hands dirty and use it as soon as I could, it's about time! 😆